openfisca_uk/calibration/losses/loss_calculator.py

```diff
@@ -98,3 +81,24 @@ def compute_loss(
                 for entry in loss_category_log
             ]
         return loss
+
+    @staticmethod
+    def create_k_fold_cv_calculators(sim: Microsimulation, k: int = 1) -> list:
+        """Creates a list of loss calculators for k-fold cross validation.
+
+        Args:
+            sim (Microsimulation): A microsimulation from which to draw demographic data.
+            k (int, optional): The number of folds (results in validation splits of 1 / k). Defaults to 1.
+
+        Returns:
+            List[LossCalculator]: The list of loss calculators.
+        """
+        base_calculator = LossCalculator(sim, validation_split=1)
+        loss_calculators = [LossCalculator(sim) for _ in range(k)]
+        for calculator in loss_calculators:
+            calculator.validation_metrics = []
+        for metric_name in base_calculator.validation_metrics:
```

Comment:
Could you add a comment asserting or describing how this assigns each metric to exactly one validation split?